src/diffusers/training_utils.py

```diff
@@ -251,6 +251,30 @@ def state_dict(self) -> dict:
             "collected_params": self.collected_params,
         }
 
+    def store(self, parameters: Iterable[torch.nn.Parameter]) -> None:
+        r"""
+        Args:
+        Save the current parameters for restoring later.
+            parameters: Iterable of `torch.nn.Parameter`; the parameters to be
+                temporarily stored.
+        """
+        self.collected_params = [param.detach().cpu().clone() for param in parameters]
```

Comment:
(nit) Why is it called `collected_params`? Might it be better to call it `temp_stored_params`?

Reply:
But it's a nit so happy to merge without